src/app/components/navbar/navbar.component.html

```diff
@@ -1,4 +1,4 @@
-<nav class="navbar bg-dark fixed-top bg-light custom">
```

Comment:
Why was this needed?

Reply:
Having the navbar as fixed-top made our content begin at the top of the page and underneath the navbar (therefore hiding it from view), rather than at the base of it.